scss/_base.forms.scss

```diff
@@ -108,6 +122,11 @@
     text-indent: .01px;
     text-overflow: '';
 
+    &:active {
+      outline: 1px solid $color-focus;
+      outline-offset: 2px;
+    }
```

Comment:
Seems like a bit of duplication here on the `:active` could this be added to the focus mixin?